docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -273,11 +272,14 @@ The ASE-generated plot looks like this:
 
 ![NEB Plot](../../img/neb.png)
 
+*[Do the green lines have any significance?]*
+
 The barrier we obtained is about 0.23 eV, which compares well with the value of 0.25 eV (24.2 kJ/mol) [from Wikipedia](https://en.wikipedia.org/wiki/Pyramidal_inversion#Energy_barrier).
 
 We might also notice the slight difference in energy between the two end-points of the path (about 1 meV). These should be identical, so any difference can be used as a very rough guide for the amount of numerical noise in the geometry optimisations.
+*[Surely this is more about the initial calculations rather than the NEB? Is this noise figure useful?]*
```

Comment:
I'd say that the noise in the end points (from the initial geom optimisations as you say), gives you a rough handle how converged the calculations are. So the noise on any subsequent geometry optimisations (including the subsequent NEB calculation in this case)  should be a similar magnitude, unless you change your parameters (XC, basis set, pseudo, force tolerance etc.) of course. 

If equivalent structures (the end points in this example) were very different in energy, it would suggest you need to tighten up your convergence criteria or force/energy tolerances etc.

Reply:
Again, I think your expanded explanation helps the reader under the significance of the point you are making.

Reply:
```suggestion
If equivalent structures (the end-points in this example) were found to have very different energies, you would need to go back and tighten up your convergence criteria or force/energy tolerances in the initial geometry optimisations and re-run the NEB calculation with these tightened parameters. 
```